### DISCUSSION TITLE
help

### DISCUSSION THREAD

**Discussion by SMBEN** [mtex-toolbox/mtex#1404] [Ask Anything]
Hi 
I try to use a rotation by Axis but I can't do that! 
I used Euler Angle  rotation and I succeeded by using: rot=rotation('Euler', 90*degree, 0*degree, 0*degree);
What exactly do I need to write in order to do a rotation 90 degrees along the X-axis ??
thank you

**Comment by zmichels** [comment]
What you used with Euler angles should work. But if you want to use the axis+angle approach for definition, you could try the following:

rot = rotation(‘axis’,vector3d.X,’angle’,90*degree)

**Reply by SMBEN** [reply]
Sorry, it does not work !
![2022-04-19_031905](https://user-images.githubusercontent.com/99767925/163906460-b12ebaa6-d76d-4edd-8a55-350787969294.jpg)

**Reply by kilir** [reply]
Have a look at your syntax highlighting in Matlab. Those schoukd be `'` instead of ` `` `. 
Also  by now, you can use `orientation.byAxisAngle(axis, angle)`.
Cheers, Rüdiger
